src/modules/BootStorage.rb

```diff
@@ -27,36 +27,12 @@ module Yast
   class BootStorageClass < Module
     include Yast::Logger
 
-    # Disk where to place MBR code. By default one with /boot partition
-    # @return [Y2Storage::Disk]
-    def mbr_disk
+    # Moint point for /boot. If there is not separated /boot, / is used instead.
```

Comment:
When I hear "mount point" I think of a filesystem path, a String. But this method returns a Filesystem, whatever that is. All callers assign it to a variable named `fs`. Rename it to `boot_filesystem`?

Reply:
yes, makes sense. I start with this one and later prefer using filesystem instead.

Reply:
(done)